rtl/include/eclhelper.hpp

```diff
@@ -48,6 +48,9 @@ typedef unsigned short UChar;
 
 //Should be incremented whenever the virtuals in the context or a helper are changed, so
 //that a work unit can't be rerun.  Try as hard as possible to retain compatibility.
+
+//TBD - I think the only thing that might break with my changes if if someone compiles a query using FORMENCODED HTTPCALL and the rolls the build back to a previous one.
+
```

Comment:
Temporary comment to get attention.  As if the automated test warnings weren't enough.

Reply:
Removed comment.